sample_apps/rust/examples/utils/timestream_helper.rs

```diff
@@ -0,0 +1,136 @@
+use aws_sdk_timestreamwrite as timestream_write;
+use aws_types::region::Region;
+use clap::Parser;
+
+static DEFAULT_DATABASE_NAME: &str = "devops_multi_sample_application";
+static DEFAULT_REGION: &str = "us-east-1";
+static DEFAULT_TABLE_NAME: &str = "host_metrics_sample_application";
+
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+pub struct Args {
+    // The Timestream for LiveAnalytics database name to use for all queries
+    #[arg(short, long, default_value = DEFAULT_DATABASE_NAME)]
+    pub database_name: String,
+
+    // The name of the AWS region to use for queries
+    #[arg(short, long, default_value = DEFAULT_REGION)]
+    pub region: String,
+
+    // The Timestream for LiveAnalytics table name to use for all queries
+    #[arg(short, long, default_value = DEFAULT_TABLE_NAME)]
+    pub table_name: String,
+}
+
+pub async fn get_connection(
+    region: &String,
+) -> Result<timestream_write::Client, timestream_write::Error> {
+    let config = aws_config::defaults(aws_config::BehaviorVersion::latest())
+        .region(Region::new(region.to_owned()))
+        .load()
+        .await;
+    let (client, reload) = timestream_write::Client::new(&config)
+        .with_endpoint_discovery_enabled()
+        .await
+        .expect("Failed to get connection to Timestream");
+
+    tokio::task::spawn(reload.reload_task());
+    Ok(client)
+}
+
+pub async fn create_database(args: &Args) -> Result<(), timestream_write::Error> {
+    let client = get_connection(&args.region).await?;
+    println!("Creating new table {:?}", args.table_name);
+
+    client
+        .create_database()
+        .set_database_name(Some(args.database_name.to_owned()))
+        .send()
+        .await?;
+
+    Ok(())
+}
+
+pub async fn create_table(args: &Args) -> Result<(), timestream_write::Error> {
+    let client = get_connection(&args.region).await?;
```

Comment:
Would this mean a new client and Timestream connection is created for each call to `create_table` and `create_database`? Maybe we should instead use one client and pass it to each function. [The goV2 `timestream-helper` handles this by making the helper a struct with clients as fields](https://github.com/Bit-Quill/amazon-timestream-tools/blob/217167baeadd7dc21b11bd23af8797f9e9d8252a/sample_apps/goV2/utils/timestream-helper.go#L27-L54).

Reply:
Yes this is better. I noticed it as well and already fixed.